pkg/provisioner/templates/container-toolkit.go

```diff
@@ -34,7 +34,7 @@ curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dear
   && \
     sudo apt-get update
 
-sudo apt-get install -y nvidia-container-toolkit
+install_packages_with_retry nvidia-container-toolkit
 
 # Configure container runtime
 sudo nvidia-ctk runtime configure --runtime={{.ContainerRuntime}} --set-as-default
```

Comment:
One thing that should be mentioned is that if we add `--enable-cdi` as an argument here, then we don't NEED containerd v2.0. Would it be simpler to expose that as an argument instead of reworking the containerd installation?

Not that I don't think having access to containerd is valuable, just that it would simplify unblocking DRA testing.

Reply:
Yeah it would be a much smaller and simpler PR, do you think we are too far from agreeing on this one?

Reply:
I have created https://github.com/NVIDIA/holodeck/pull/319 
I will leave this PR for a further release, I'll cut a release once we merge #319